src/UI/Shared/WB.UI.Shared.Enumerator/CustomBindings/ImageViewBitmapWithFallbackBinding.cs

```diff
@@ -61,7 +61,7 @@ protected virtual void SetDefaultImage(ImageView control)
 
     protected virtual void SetupImageView(ImageView control, DisplayMetrics displayMetrics, BitmapFactory.Options boundsOptions)
     {
-        
+

```

Comment:
use in this binding class IImageHelper

Reply:
Either it becomes too simple - one line code, or signature too complex  with action SetupImageView passing into the method - that we agreed to avoid.

Reply:
IImageHelper has the same code for resizing, use it